code/modules/mob/mob_defines.dm

```diff
@@ -46,6 +46,7 @@
 	var/cultslurring = 0	//Carbon
 	var/real_name = null
 	var/druggy = 0			//Carbon
+	var/disgust = 0 		//Carbon
```

Comment:
Please don't repeat this pattern. If it's carbon put in on carbon level. Or ideally make this status effect.